hot-reload.js

```diff
@@ -13,9 +15,15 @@ const filesInDirectory = dir => new Promise (resolve =>
     )
 )
 
-const timestampForFilesInDirectory = dir =>
-        filesInDirectory (dir).then (files =>
-            files.map (f => f.name + f.lastModifiedDate).join ())
+const timestampForFilesInDirectory = dir => {
+        return filesInDirectory (dir).then (files => {
+            if (includes.length && !includes.every(i => files.some(f => i === f.name))) return;
```

Comment:
Can you please clarify what that line does?

Reply:
@xpl It checks if the user has specified any required `includes`, and if so, checks if not all of the files specified for inclusion actually exist in this timeout cycle - if any of the specified includes don't exist, it short-circuits so that we will wait for the next cycle to check again, hoping all the required files will exist then.